cs/unittest/TestMarshalNumeric.cs

```diff
@@ -1,4 +1,4 @@
-
+﻿
```

Comment:
Change intended?

Reply:
not intented

Reply:
i think this file shouldn't even be part of the repo

Reply:
https://github.com/VowpalWabbit/vowpal_wabbit/pull/2492